application/aggregator.py

```diff
@@ -0,0 +1,58 @@
+"""
+Aggregates the data from all databases into one CSV with minimal cutting/processing
+"""
+import sqlite3
+import csv
+import pandas as pd
+
+HEADERS = list(['track_id', 'title', 'song_id', 'release', 'artist_id', 'artist_mbid',
+                'artist_name', 'duration', 'artist_familiarity', 'artist_hotttnesss',
+                'year', 'lyrics'])
+
+DATA = []
```

Comment:
Can we make this thing pep8 compliant? https://www.python.org/dev/peps/pep-0008/
With global variables for internal use prefixed with an _underscore. Also Headers is constant, but data isn't why is data all caps? 

Since you don't have fancy things like const in python this naming stuff is pretty important to communicate intent.

Reply:
I was going off of what pylint was saying. I'm guessing it's because this isn't in a method so it thinks they all should be constant? I can add an exception to pylint or figure out how to set up the file so it looks for the right convention